lib/pubsub/subscription.js

```diff
@@ -401,7 +401,7 @@ Subscription.prototype.pull = function(options, callback) {
  */
 Subscription.prototype.setAckDeadline = function(options, callback) {
   var body = {
-    ackId: options.ackId,
+    ackIds: util.arrayize(options.ackIds),
```

Comment:
Yeah, we just have to leave a note on the next release.

On Thursday, July 9, 2015, Takashi Matsuo <notifications@github.com> wrote:

> In lib/pubsub/subscription.js
> <https://github.com/GoogleCloudPlatform/gcloud-node/pull/711#discussion_r34309733>
> :
>
> > @@ -401,7 +401,7 @@ Subscription.prototype.pull = function(options, callback) {
> >   */
> >  Subscription.prototype.setAckDeadline = function(options, callback) {
> >    var body = {
> > -    ackId: options.ackId,
> > +    ackIds: util.arrayize(options.ackIds),
>
> Nit:
> I think you break the compatibility here, but I don't think it's a big
> deal.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-node/pull/711/files#r34309733>
> .
>